qv2ray.patch

```diff
@@ -0,0 +1,34 @@
+diff --git a/Qv2ray.pro b/Qv2ray.pro
+index 98ec27e..c251834 100644
+--- a/Qv2ray.pro
++++ b/Qv2ray.pro
+@@ -232,24 +232,24 @@ unix {
```

Comment:
Imho, this should be fixed upstream and just make it use the $prefix.

Reply:
We add an [argument](https://github.com/lhy0403/Qv2ray/commit/2d212cd1b6fbd67537cb940cf482df888f6bd1ea) to do that.